lib/client/mfa.go

```diff
@@ -145,7 +139,7 @@ func PromptMFAChallenge(ctx context.Context, proxyAddr string, c *proto.MFAAuthe
 		}
 	}
 	return nil, trace.BadParameter(
-		"failed to authenticate using all U2F and TOTP devices, rerun the command with '-d' to see error details for each device")
+		"failed to authenticate using all MFA and TOTP devices, rerun the command with '-d' to see error details for each device")
```

Comment:
Isn't TOTP technically an MFA device too?

Reply:
True, should I drop the "and TOTP" part of it?

Reply:
I would.

Reply:
Done, changed to `failed to authenticate using all MFA devices, (...)`.